### ISSUE TITLE
BGs mismatched in 54 - Balcony

### ISSUE THREAD

**Issue by Kayael76** [MoArtis/ResidentEvilSeamlessHdProject#99]
It's gonna be hard to follow. Some BG involving the projector plugged or unplugged are mixed up.

R30B01 should be R30B18 (and vice-versa)
R30B1A should be R30B13

The GC output is thus mixed up too. For some reason reason another output is mixed though the PC upscales and rips are correctly matched so here's what's supposed to be good :

Good tex1_640x480_764ed0f4b7da98fb_5n (good R30B18)
![tex1_640x480_764ed0f4b7da98fb_5](https://user-images.githubusercontent.com/48800201/59470955-b86de280-8e39-11e9-8414-647bd07e52fe.png)

Good tex1_640x480_fac1e2e4efd26ff5_5 (good R30B01)
![tex1_640x480_fac1e2e4efd26ff5_5](https://user-images.githubusercontent.com/48800201/59471630-df2d1880-8e3b-11e9-9776-1f34020a8dd2.png)

Good tex1_640x480_b8297379d2a2bf31_5 (good R30B19)
![tex1_640x480_b8297379d2a2bf31_5](https://user-images.githubusercontent.com/48800201/59470994-e3583680-8e39-11e9-8f56-e7dc89a0b4bc.png)

Good tex1_640x480_520c95a4f90cf65f_5 (good R30B02)
![tex1_640x480_520c95a4f90cf65f_5](https://user-images.githubusercontent.com/48800201/59471496-878ead00-8e3b-11e9-82f3-021945da81dd.png)

Good tex1_640x480_bb068247093474fb_5 (Good R30B13)
![tex1_640x480_bb068247093474fb_5](https://user-images.githubusercontent.com/48800201/59471074-3cc06580-8e3a-11e9-8300-1729d2059718.png)

**Comment by Kayael76** [comment]
Ok everything has been renamed accordingly so you can eventually reprocess those 5 BG :)

**Comment by MoArtis** [comment]
New info:
- R30B1A and R30B13 's BGS are identical
- BUT R30B1A has no mask GC name matched to it. (is it unused?)

Everything else should be testable and good to know. If you notice the missing mask of R30B1A, create a new issue for clarity. Thx!

**Comment by Kayael76** [comment]
R30B1A is unused, you'll notice the plug is out compared to R30B13, this instance in never used in game (R30B13 is displayed when you encounter Nemesis but you can't unplug the projector prior to this event). 

TLDR, it should work correctly now !

**Comment by Kayael76** [comment]
Ah shit no, R30B13 is still mismatched : 

![R30B13](https://user-images.githubusercontent.com/48800201/60470920-8f838500-9c62-11e9-9d03-daf74e6fb1cc.jpg)

Everything else works fine.

**Comment by MoArtis** [comment]
Oh So they are not identical 😂

**Comment by MoArtis** [comment]
I need to stop reading half of people wrote me. I miss that you said that R30B1A is unused. It's easy to fix then. ;)

**Comment by MoArtis** [comment]
![GLEE08-82](https://user-images.githubusercontent.com/3904610/60645458-49e5d880-9e6b-11e9-8166-b1c72aa6751f.png)

Done!
